frontend/src/metabase/static-viz/components/RowChart/theme.ts

```diff
@@ -0,0 +1,36 @@
+import { ColorGetter } from "metabase/static-viz/lib/colors";
+import { RowChartTheme } from "metabase/visualizations/shared/components/RowChart/types";
+
+export const getStaticChartTheme = (getColor: ColorGetter): RowChartTheme => {
```

Comment:
I like how you define everything style related to the static chart in a single file that's so easy to manage, I should have done this for the gauge chart too, as now the values are spread across different files afaik.